content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -108,8 +104,7 @@ const tracer = require('dd-trace').init({
   logger: {
     debug: message => logger.trace(message),
     error: err => logger.error(err)
-  },
-  debug: true
+  }
```

Comment:
The above paragraph still has instructions using the old `debug` configuration.

Reply:
I have removed this reference